File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/community/CommunityAction.kt
package com.hhp227.concafe.presentation.community

sealed interface CommunityAction {
    data object Refresh : CommunityAction
    data object LoadMore : CommunityAction
    data object ClickWritePost : CommunityAction
    data class ClickPost(val postId: String) : CommunityAction
    data object DismissError : CommunityAction
}

File: iosApp/iosApp/Presentation/Community/CommunityAction.swift
//
//  CommunityAction.swift
//  ConCafe
//
//  Created by 홍희표 on 4/27/26.
//

import Foundation

enum CommunityAction {
    case refresh
    case loadMore
    case clickWritePost
    case clickPost(postId: String)
    case dismissError
}
